bin/requirements.txt

```diff
@@ -7,3 +7,4 @@ PyYAML>=3.0
 stop-words
 tqdm
 texsoup<=0.1.4
+nltk
```

Comment:
Why do we now depend on nltk?  I couldn't find a matching new import.

Reply:
It's used in `bin/fixedcase/common.py`.